### PR TITLE
Move catalog-oauth image to gcr.io/gcp-services

### PR DESCRIPTION
**Pull request by martinmaly**
- update template with the new image location
- re-generate templates.go

### CODE REVIEW

**Comment by martinmaly** [comment]
Note that this PR also contains #137 in it. GitHub cannot keep dependent PRs straight :(

File: installer/templates/gcp/google-oauth-deployment.yaml.tmpl [hunk 1]
```diff
@@ -41,7 +41,7 @@ spec:
       serviceAccountName: "controller-manager"
       containers:
       - name: catalog-oauth
-        image: gcr.io/seans-sandbox/catalog-oauth:latest
+        image: gcr.io/gcp-services/catalog-oauth:latest
```
Comment (droot) [inline installer/templates/gcp/google-oauth-deployment.yaml.tmpl]: Nice.

**Comment by maqiuyujoyce** [comment]
I also found "seans-sandbox" in [this md file](https://github.com/GoogleCloudPlatform/k8s-service-catalog/blob/master/build-service-catalog.md). Is it an outdated file that we don't care about?

**Comment by martinmaly** [comment]
It is in many places - readmes, some test scripts etc. This file actually controls the `sc add-gcp-broker` command. At some point we'll need to clean up the rest of it...

**Pull request merged**